File: project3/Team3/projectBQ/src/main/java/bitc/full502/projectbq/dto/StockDto.java
package bitc.full502.projectbq.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class StockDto {

//    Stock 정보 "20250918 완료"

    private Long id;
    private String itemName;
    private String itemCode;
    private String manufacturer;
    private Long quantity;
    private String categoryName;
    private String warehouseName;
    private String warehouseKrName;
}

File: project3/Team3/App_BQ/app/src/main/java/bitc/full502/app_bq/data/model/StockDto.kt
package bitc.full502.app_bq.data.model


data class StockDto(
    val id: Long,
    val itemName: String,
    val itemCode: String,
    val manufacturer: String,
    val quantity: Long,
    val categoryName: String,
    val warehouseName: String,
    val warehouseKrName: String
)